extensions/amp-story-shopping/0.1/test/test-amp-story-shopping-config.js

```diff
@@ -196,15 +217,26 @@ describes.realWin(
     });
 
     it('does use inline config when remote src is invalid', async () => {
-      env.sandbox.stub(Services, 'xhrFor').returns({
-        fetchJson() {
-          throw new Error();
-        },
-      });
+      await createAmpStoryShoppingConfig();
+      const exampleURL = 'invalidRemoteURL';
+      pageElement.setAttribute('src', exampleURL);
       const result = await createAmpStoryShoppingConfig('invalidRemoteUrl');
       expect(result).to.deep.eql(keyedDefaultInlineConfig);
     });
 
+    it('test invalid remote config url', async () => {
+      await createAmpStoryShoppingConfig();
+      const exampleURL = 'invalidRemoteURL';
+      pageElement.setAttribute('src', exampleURL);
+      expectAsyncConsoleError(async () => {
+        expect(async () => {
+          await getShoppingConfig(pageElement);
```

Comment:
This test is good: you call the public API and write "expect" statements for what comes out of it

Reply:
I have refactored the tests to use public APIs - we should be good to go on these ones now.